tooling/cli/src/helpers/framework.rs

```diff
@@ -21,6 +23,8 @@ pub enum Framework {
 impl Framework {
   pub fn dev_path(&self) -> String {
     match self {
+      Self::SolidJS => "http://localhost:5173",
```

Comment:
I just tried `npx degit solidjs/templates/js my-app` and it runs on localhost:3000

Reply:
Yeah but I assume the official template is more used, if so we should go with that one.

Reply:
I think this issue is related even to few other frameworks. I think the best way to solve this problem is to read the bundler configuration file. I will try to solve it in the next few days 😊